shared/src/commonTest/kotlin/com/kcc/kmmhackathon/shared/network/LocationUtilTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.kcc.kmmhackathon.shared.network
+
+import com.kcc.kmmhackathon.shared.utility.LocationUtil
+import kotlin.math.round
```

Comment:
Is this used?

Reply:
Not since I moved it into the extension - will remove import